src/state/lists/updater.ts

```diff
@@ -75,20 +81,26 @@ export default function Updater(): null {
             break
 
           case VersionUpgrade.MAJOR:
-            dispatch(
-              addPopup({
-                key: listUrl,
-                content: {
-                  listUpdate: {
-                    listUrl,
-                    auto: false,
-                    oldList: list.current,
-                    newList: list.pendingUpdate
-                  }
-                },
-                removeAfterMs: null
-              })
-            )
+            ReactGA.event({
+              category: 'Lists',
+              action: 'Update List from Popup',
+              label: listUrl
+            })
+            dispatch(acceptListUpdate(listUrl))
```

Comment:
While this makes total sense for our hosted tokenlists, users who added their own token list will likely not want to auto accept updates? Do you think it would be better to only auto accept MAJOR version updates if the tokenlist url is included in one we host (`src/constants/lists.ts#DEFAULT_LIST_OF_LISTS`)?

Reply:
okay sure @bmino